.ci/jenkins/release-project-dependencies.yaml

```diff
@@ -0,0 +1,8 @@
+version: "2.1"
```

Comment:
I think you added this file in the wrong place, it should be under `.ci` and not `.ci/jenkins`, right?